app/components/Home.tsx

```diff
@@ -21,6 +21,15 @@ const MainBrowserWindow: React.FunctionComponent<Props> = (props: Props) => {
   const [url, updateUrl] = useState<string>('');
   const [title, updateTitle] = useState<string>('');
   const [modal, setModal] = useState<boolean>(false);
+  const [value, setValue] = useState(localStorage.getItem('tabtitle'));
+
+  useEffect(() => {
+    let getList = localStorage.getItem('tabtitle');
+    getList = JSON.parse(getList);
+    if(getList){
+      title: getList
+    }
+  });
```

Comment:
I am not able to make sense it @yashkapoor8710 can you explain why we have added this.

Reply:
I am working on Restore the Previous Tabs so it is just a test code to show the tabs on reload